generative_ai/embeddings/document_retrieval_example.py

```diff
@@ -19,27 +19,41 @@
 
 
 def embed_text() -> list[list[float]]:
-    """Embeds texts with a pre-trained, foundational model.
+    """Calculates embeddings for the given text inputs.
 
     Returns:
-        A list of lists containing the embedding vectors for each input text
+        A list of lists containing the embedding vectors for each input text.
     """
 
-    # A list of texts to be embedded.
-    texts = ["banana muffins? ", "banana bread? banana muffins?"]
+    model = TextEmbeddingModel.from_pretrained("text-embedding-004")
```

Comment:
I'm not sure how/why this works without the standard call to `vertexai.init(...)` (but it works 🤷 ). Should I probably add this call to be future-proof?

Reply:
`vertexai.init` is a must for a first time setup. Doing it again is harmless but good to have option. So this is not a bug.